index.html

```diff
@@ -0,0 +1,274 @@
+      <ul class="catalog-container">
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="lang" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+      </ul>
+      <a href="#" class="catalog-btn">join the tech wolrd</a>
+      <a href="#" class="catalog-text">see the whole program</a>
+    </section>
+
+    <!-- catalog-section-ends -->
+
+    <!-- student-review-start -->
+    <section class="partners" id="testimonial">
+      <h1 class="testimonial-title">Students Testimonial</h1>
+      <hr class="line" />
+      <div class="testimonials"></div>
+    </section>
+
+    <div class="partners-container">
+      <h2 class="partner-title">Take the leap of Faith</h2>
+      <p class="partner-description">
+        its takes time, patience, perseverance and dedication to learn a coding
+        language. put in 2 hours a day into learning something new and you'll be
+        a pro in no time.
+      </p>
+      <div class="partners-buttons">
+        <button type="button">Start Now</button>
+        <button type="button">Have questions ?</button>
+      </div>
+    </div>
+
+    <!-- student-review-end -->
+
+    <!-- footer-section-starts -->
+    <footer class="footer-container">
+      <div class="partners-img">
+        <ul class="partners-images">
+          <li><img src="./images/tesla-128.webp" alt="image5" /></li>
+          <li>
+            <img
+              src="./images/Apple_and_Google_Partner_.webp"
+              class="google"
+              alt="image1"
+            />
+          </li>
+          <li><img src="./images/youtube.webp" alt="image2" /></li>
+          <li><img src="./images/windows.webp" alt="image3" /></li>
+          <li>
+            <img src="./images/skype-128.webp" class="skype" alt="image4" />
+          </li>
+        </ul>
+      </div>
```

Comment:
- [ ] Kindly make sure that the footer section has similar spacing, borders, shadows, paddings, and alignments than the orignal design.
- [ ] Also the homepage footer has a white background and that of the about page has a dark background. Kindly fix it.
Your design
![Capture3](https://user-images.githubusercontent.com/85095702/175773634-adabf797-39b9-4817-aa4f-d26c9c9dec56.jpg)
![Capture4](https://user-images.githubusercontent.com/85095702/175773659-7ed43e17-b40c-4d39-9b96-3407c86c0fec.jpg)
Behance template
![figma3](https://user-images.githubusercontent.com/85095702/175773675-4c207de0-8f99-4f42-97ff-8484803322bc.jpg)